lib/session.ts

```diff
@@ -52,6 +52,29 @@ export class Session extends Entity {
     return this._session.error;
   }
 
+  /**
+   * Returns the unique identifier for the session in the format:
+   * "local_<number>-remote_<number>-<connection-id>" or an empty string if the local channel or
+   * remote channel are not yet defined.
+   */
+  get id(): string {
+    let result: string = "";
+    const session: any = this._session;
+    if (session.local && session.local.channel) {
+      result += `local_${session.local.channel}-`;
+    }
+
+    if (session.remote && session.remote.channel) {
+      result += `remote_${session.remote.channel}-`;
+    }
+
+    if (result) {
+      result += `${this._connection.id}`;
+    }
+
+    return result;
```

Comment:
@grs - does this format look good?

Reply:
Can you also review the changes in the PR?

Reply:
Yes, looks good to me.